lib/_http_outgoing.js

```diff
@@ -119,6 +121,25 @@ Object.defineProperty(OutgoingMessage.prototype, 'writableFinished', {
   }
 });
 
+Object.defineProperty(OutgoingMessage.prototype, 'writableObjectMode', {
+  get: function() {
```

Comment:
Suggestion: Can you use the more common shorthand notation? (I’ve also noticed this in other PRs, I didn’t point it out because it doesn’t really matter, but if we add more and more of these it would be good to keep an eye on consistency a bit more often.)

```suggestion
  get() {
```